coc.md

```diff
@@ -32,7 +32,6 @@ layout: page
 					<li>Threats of violence</li>
 					<li>Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm</li>
 				</ul>
-			<p>Jokes that resemble such offensive comments are still considered offensive even if meant ‘ironically’.</p>
```

Comment:
Hmm. I'd actually deliberately moved this down -- I do want to call racism out specifically, but it might actually apply generally. Thoughts?